dir.props

```diff
@@ -19,6 +19,10 @@
     <DotNetCliToolDir>$(ProjectDir)Tools/dotnetcli/</DotNetCliToolDir>
     <PatchesDir>$(ProjectDir)patches/</PatchesDir>
     <PackagesDir>$(ProjectDir)packages/</PackagesDir>
+    <ReferencePackagesStagingDir>$(ProjectDir)reference-packages/staging/</ReferencePackagesStagingDir>
```

Comment:
I think the tarball production build should put this stuff in `bin` if possible.